src/clr-angular/data/datagrid/datagrid-pagination.spec.ts

```diff
@@ -309,6 +312,38 @@ export default function(): void {
         expect(invalidButton).toBeUndefined();
       });
     });
+
+    describe('Accessibility', function() {
+      // Until we can properly type "this"
+      let context: TestContext<ClrDatagridPagination, FullTest>;
+      let commonStrings: ClrCommonStringsService;
+
+      beforeEach(function() {
+        context = this.create(ClrDatagridPagination, FullTest, [Page, StateDebouncer]);
+        commonStrings = new ClrCommonStringsService();
+
+        context.testComponent.size = 10;
+        context.testComponent.total = 100;
+        context.testComponent.current = 10;
+        context.detectChanges();
+      });
+
+      it('expect buttons to have the correct aria-label from ClrCommonStringsService', function() {
+        const firstPage = context.clarityElement.querySelector('.pagination-first');
```

Comment:
nit - seems unnecessary to define all of these variables for one use.

Reply:
I had even more crazy idea - but for readability, I did not commit it.

```js
[
  ['.pagination-first', 'firstPage'],
  ['.pagination-last', 'lastPage'], 
  // ...
].forEach((button) => {
  expect(context.clarityElement.querySelector(button[0]).attributes['aria-label'].value)
  .toBe(commonStrings[button[1]])
})
```

I also don't like creating `alias/shortcut` variables - but the lines become too long and hard to fallow.
```js
// Example
expect(context.clarityElement.querySelector('.pagination-first').attributes['aria-label'].value).toBe(commonStrings.firstPage);
```

Another idea will be to just write it like that...

```js
const { querySelector } = context.clarityElement;
expect(querySelector('.pagination-first').attributes['aria-label'].value).toBe(commonStrings.firstPage);
// ...
```